app/components/WelcomePanel/WelcomePanel.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  Drawer,
+  DrawerCloseButton,
+  DrawerContent,
+  DrawerOverlay,
+  Flex,
+  Hide,
+  IconButton,
+  Show,
+} from "@nycplanning/streetscape";
+import { WelcomeContent, WelcomeHeader } from ".";
+import { InfoIcon } from "@chakra-ui/icons";
+import { useState } from "react";
+
+export function WelcomePanel() {
+  const [isMobileExpanded, setIsMobileExpanded] = useState(true);
+
+  const toggleMobileExpanded = () =>
+    setIsMobileExpanded((isMobileExpanded) => !isMobileExpanded);
+  return (
+    <>
+      <Show above="lg">
+        <Accordion
+          allowToggle
+          borderBottomRadius={"base"}
+          padding={{ base: 3, lg: 4 }}
+          background={"white"}
+          width={{ base: "full", lg: "21.25rem" }}
+          maxW={{ base: "21.25rem", lg: "unset" }}
+          boxShadow={"0px 8px 4px 0px rgba(0, 0, 0, 0.08)"}
+          borderTopWidth={"1px"}
+          borderTopColor={"gray.50"}
+          defaultIndex={0}
+        >
+          <AccordionItem border="none">
+            <AccordionButton aria-label="Toggle welcome panel" px={0}>
+              <WelcomeHeader />
+              <AccordionIcon />
+            </AccordionButton>
+            <AccordionPanel pb={0} px={0}>
+              <WelcomeContent />
+            </AccordionPanel>
+          </AccordionItem>
+        </Accordion>
+      </Show>
+      <Hide above="lg">
+        <Box width={"21.25rem"} paddingTop={3}>
+          <IconButton
+            aria-label="information about the capital planning portal"
+            icon={<InfoIcon boxSize={5} />}
+            borderColor={"white"}
+            borderStyle="solid"
+            borderWidth="5px"
+            borderRadius={"100%"}
+            color="black"
+            minHeight="auto"
+            minWidth="auto"
+            height="fit-content"
+            width="fit-content"
+            backgroundColor={"white"}
+            _hover={{
+              border: "none",
+              backgroundColor: "blackAlpha.100",
+            }}
+            onClick={toggleMobileExpanded}
+          />
+          <Drawer
+            placement="bottom"
+            isOpen={isMobileExpanded}
+            onClose={toggleMobileExpanded}
+          >
+            <DrawerOverlay />
+            <DrawerContent overflowY="scroll" borderTopRadius={"12px"}>
```

Comment:
<img width="541" alt="image" src="https://github.com/user-attachments/assets/60ecf831-a655-4540-9d9e-869f36e1c69c">
I get this empty bar on the right side of the drawer when I make the window narrow enough on desktop. Changing this to `overflowY="auto"` should hopefully resolve that without breaking anything else.

Reply:
change pushed. I'll leave it for you to resolve when the behavior is confirmed gone.